apps/prairielearn/src/lib/editors.ts

```diff
@@ -1162,28 +1162,49 @@ export class QuestionModifyEditor extends Editor {
 }
 
 export class QuestionDeleteEditor extends Editor {
-  private question: Question;
+  private questions: Question[];
 
-  constructor(params: BaseEditorOptions) {
+  constructor(params: BaseEditorOptions & { questionBatch?: Question[] }) {
```

Comment:
Three notes:

- This shouldn't be an optional property.
- Building the API with a single property makes this simpler for callers.
- You'll want to update the existing usage to pass `res.locals.question` in the `questions` property.

```suggestion
  constructor(params: BaseEditorOptions & { questions: Question | Question[] }) {
```

Reply:
Switched in 253c7cd80e2a8b32adc02bcbc9c16c6e7167eb18. Didn't want to change the interface but TBH, it's only used in one place, so was easy fix